gr-digital/grc/digital_cpmmod_bc.block.yml

```diff
@@ -37,4 +38,20 @@ templates:
     imports: from gnuradio import gr, digital, analog
     make: digital.cpmmod_bc(${type}, ${mod_index}, ${samples_per_symbol}, ${L}, ${beta})
 
+cpp_templates:
+    includes: ['#include <gnuradio/digital/cpmmod_bc.h>']
+    declarations: 'digital::cpmmod_bc::sptr ${id};'
+    make: |-
+        this->${id} = digital::cpmmod_bc::make(
+            ${type},
+            ${mod_index},
+            ${samples_per_symbol},
+            ${L},
+            ${beta});
+    link: ['gnuradio-digital']
+    translations:
+        analog.cpm: 'cpm('
```

Comment:
I'm afraid this won't work. You can find the parameters in the header file: https://github.com/gnuradio/gnuradio/blob/7074f054c25a69392e7ae4e1f0204a1c5ce242f8/gr-digital/include/gnuradio/digital/cpmmod_bc.h#L73